railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -28,15 +28,17 @@ ruby <%= "'#{RUBY_VERSION}'" -%>
 
 <% if depend_on_bootsnap? -%>
 # Reduces boot times through caching; required in config/boot.rb
-gem 'bootsnap', '>= 1.1.0', require: false
+gem 'bootsnap', '>= 1.4.0', require: false
 
 <%- end -%>
 <%- if options.api? -%>
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 
 <%- end -%>
-<% if RUBY_ENGINE == 'ruby' -%>
+<% if RUBY_ENGINE == "ruby" -%>
+gem "zeitwerk", ">= 1.0.0"
```

Comment:
All the rest of this file uses single quotes

Reply:
But our style guidelines have changed over time, saw an opportunity to update that one I was touching.

True that the file should be consistent.